docs/articles_en/openvino-workflow/model-optimization-guide/weight-compression.rst

```diff
@@ -6,38 +6,38 @@ LLM Weight Compression
    :hidden:
 
    weight-compression/microscaling-quantization
+   weight-compression/4-bit-weight-quantization
 
 
-Weight compression is a technique for enhancing the efficiency of models,
-especially those with large memory requirements. This method reduces the model's
+Weight compression enhances the efficiency of models,
+which have large memory requirements. This method reduces the model's
 memory footprint, a crucial factor for Large Language Models (LLMs).
 
 Unlike full model quantization, where weights and activations are quantized,
 weight compression in `Neural Network Compression Framework (NNCF) <https://github.com/openvinotoolkit/nncf>`__
-only targets the model's weights. This approach allows the activations to remain as
+only targets the model's weights. This approach enables the activations to remain as
 floating-point numbers, preserving most of the model's accuracy while improving its
-speed and reducing its size.
+speed and reducing its size. The reduction in size is especially noticeable with larger models.
+For instance the 7 billion parameter Llama 2 model can be reduced
+from about 25GB to 4GB using 4-bit weight compression.
 
-The reduction in size is especially noticeable with larger models,
-for instance the 7 billion parameter Llama 2 model can be reduced
-from about 25GB to 4GB using 4-bit weight compression. With smaller models (i.e. less
-than 1B parameters), weight compression may result in more accuracy reduction than
-with larger models.
+.. note::
+
+   With smaller language models (i.e. less than 1B parameters), weight
+   compression may result in more accuracy reduction than with larger models.
+   Therefore, weight compression is recommended for use with LLMs only.
 
 LLMs and other models that require
 extensive memory to store the weights during inference can benefit
 from weight compression as it:
```

Comment:
Maybe we don't need this piece. It just restates stuff. But we need to check what "other models" means in this case.

Reply:
> Maybe we don't need this piece. It just restates stuff. But we need to check what "other models" means in this case.

I have changed it with the [commit](https://github.com/openvinotoolkit/openvino/pull/27432/commits/b354ec794b5d3ef1ef25ae26a61cddfa9ce6dff5)
as agreed with @AlexKoff88